src/hammer-vlsi/par/openroad/__init__.py

```diff
@@ -815,31 +848,28 @@ def detailed_route(self) -> bool:
 
         # set_thread_count [exec getconf _NPROCESSORS_ONLN]
 
-        write_verilog -include_pwr_gnd {self.output_netlist_filename}
+        # write_verilog -include_pwr_gnd {self.output_netlist_filename}
 
-        # TODO: many other arguments available
+        # NOTE: many other arguments available
         detailed_route -guide {self.route_guide_path} \\
             -bottom_routing_layer met1 \\
             -top_routing_layer met5 \\
```

Comment:
Ack, another hard-coded value. We should really make top/bottom routing layers into keys, since it's basically required in every technology to be specified. All plugins would need to be updated though (will make an issue)